core/src/main/java/jenkins/util/SystemProperties.java

```diff
@@ -376,27 +462,4 @@ public static Long getLong(String name, Long def, Level logLevel) {
         return def;
     }
 
-    @CheckForNull
-    private static String tryGetValueFromContext(String key) {
-        if (!JenkinsJVM.isJenkinsJVM()) {
```

Comment:
Inverting control. This was always a bit hackish anyway. Amends #3362 to make agent-side calls _work_, not merely not _crash_.